### PR TITLE
Add user on info output

### PR DESCRIPTION
**Pull request by tprelle**
### Description
Add user info in %%info output fixes https://github.com/jupyter-incubator/sparkmagic/issues/678

### Checklist
- [*] Wrote a description of my changes above 
- [*] Added a bullet point for my changes to the top of the `CHANGELOG.md` file
- [*] Added or modified unit tests to reflect my changes
- [*] Manually tested with a notebook
- [*] If adding a feature, there is an example notebook and/or documentation in the `README.md` file

### CODE REVIEW

**Review by devstein — COMMENTED** [review]
@tprelle Thanks for contributing! The code looks good to me. Can you fix the merge conflicts with master or give me push permission to your fork? Once the conflicts are resolved, I'll approve & merge

**Comment by tprelle** [comment]
@devstein Thanks for the review. I just fix the merge conflicts.

**Review by devstein — APPROVED** [review]

**Pull request merged**
